.github/workflows/bench-diff.yml

```diff
@@ -13,14 +13,21 @@ jobs:
     # Support for branches, non-forked-prs or forked-prs with a label
     if: |
       contains(github.event.pull_request.labels.*.name, 'safe-to-test') ||
-      github.event_name != 'pull_request' || github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false
+      github.event_name != 'pull_request' ||
+      (github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false)
```

Comment:
I'm a bit confused about this part, actually.

According to the command forked-prs are supported.

Looking at the condition, forks are excluded?

Reply:
It should run for the following events:

- push - if no a forked Pull Request
- label [safe-to-test](https://github.com/elastic/apm-data/labels/safe-to-test) - if a forked Pull Request

Reply:
Ah, I read the comment wrong.

so it's 
- branches
- non-forked-prs
- forked-prs with the "safe-to-test" label.

I see